docs/API.md

```diff
@@ -166,15 +166,17 @@ query HelloWorld {
 
 <h5 id="graphql-tags">Tags</h5>
 
-| Tag              | Description                                               |
-|------------------|-----------------------------------------------------------|
-| graphql.document | The original GraphQL document.                            |
+| Tag               | Description                                               |
+|-------------------|-----------------------------------------------------------|
+| graphql.document  | The original GraphQL document.                            |
+| graphql.variables | The variables applied to the document.                    |
 
 <h5 id="graphql-config">Configuration Options</h5>
 
-| Option  | Default                                          | Description                            |
-|---------|--------------------------------------------------|----------------------------------------|
-| service | *Service name of the app suffixed with -graphql* | The service name for this integration. |
+| Option          | Default                                          | Description                                                            |
+|-----------------|--------------------------------------------------|------------------------------------------------------------------------|
+| service         | *Service name of the app suffixed with -graphql* | The service name for this integration.                                 |
+| filterVariables | `undefined` *No variables will be recorded*      | To enable recording provide a callback. E.g. `variables => variables`. |
```

Comment:
Not sure about the naming here. This is technically no longer a filter. I also still think it should be a noun. Maybe just `variables`? Then it could eventually be either a list of variable names or a callback (just a callback for now).